examples/huggingface/README.md

```diff
@@ -0,0 +1,103 @@
+# Implementation of NLP algorithms using Zenml
```

Comment:
As the directory is called huggingface it would be nice to have a small introductory paragraph what huggingface is and how it helps.